service/RTC/StandardVideoSettings.ts

```diff
@@ -0,0 +1,47 @@
+import browser from '../../modules/browser';
+
+/**
+ * Standard scalability mode settings for different video codecs and the default bitrates.
+ */
+export const STANDARD_CODEC_SETTINGS = {
```

Comment:
Maybe a simple way to avoid the codec casing stuff is to do:

```
const AV1_SETTINGS = { ... };
export const STANDARD_CODEC_SETTINGS = {
    av1: AV1_SETTINGS,
    AV1: AV1_SETTINGS,
    ...
}

```